Formula/glib.rb

```diff
@@ -31,6 +31,7 @@ class Glib < Formula
   end
 
   on_linux do
+    depends_on "dbus"
```

Comment:
It seems odd that we don't need this in CI, though. CC @Homebrew/linux for thoughts.

Reply:
This fixes an opportunistic linking case. Users that have dbus installed on their system and try to compile glib from source need this. I'm OK with the change as-is.